test/gurka/gurka.h

```diff
@@ -566,7 +590,7 @@ findEdge(const map& map,
          const baldr::GraphId& tile_id,
          const std::string& way_name,
          const std::string& end_node) {
-  std::unique_ptr<baldr::GraphReader> reader(new baldr::GraphReader(map.config));
+  baldr::GraphReader reader(map.config.get_child("mjolnir"));
```

Comment:
which is we cant really do this. as soon as this goes out of scope at the end of this function, the memory that the edges it returns are pointing is reclaimed. in my branch i just deleted this function. i dont remember exactly but it wither wasnt used or was trivial to work arounnd in the tests that used it